source/query-with-natural-language/enable-natural-language-querying.txt

```diff
@@ -36,9 +36,15 @@ you must:
 - Sign into Atlas. For details on how to get a free Atlas 
   account and deploy a free M0 cluster, see :ref:`atlas-getting-started`.
 
-- Have Compass version 1.4 or later installed. For instructions on 
+- Have Compass version 1.4.0 or later installed. For instructions on 
```

Comment:
[suggestion] Active voice + we typically use the substitution for referencing to Compass
```suggestion
- Install |compass-short| version 1.4.0 or later. For instructions on 
```